en/codegen/layout-codegen.md

```diff
@@ -25,49 +25,112 @@ In the below image the developer would select the Sketch Groups `peopleList` and
 ### Component Sizing
 
 > [!Note]
-> Artboards in Sketch do not have fixed height or fixed width settings so this section on Component Sizing only applies to a Component withing a Group that has been selected for code generation.
+> Artboards in Sketch do not have fixed height or fixed width settings so this section on Component Sizing only applies to a Component within a Group that the user selects for code generation.
 
 When a Group is selected for component code generation and the Components inside have a fixed height and or a fixed width, those fixed size values will be added to the generated component CSS.
-Normally Angular components are sized or resized by their parent at runtime. However, there are design scenarios where the designer wants the component to be a fixed size element when added to the Angular application. Code generation supports this design scenario.
+Normally the parent of Angular components is responsible for their sizing or resizing at runtime. However, there are design scenarios where the designer wants the component to be a fixed size element when added to the Angular application. Code generation supports this design scenario.
 
-### After Component Code Generation – Before Rendering in The Browser
+## Layouts
 
-Developers need to perform the following edits of generated components before rendering in the browser:
+Sketch arranges artboards of projects through absolute positioning, using properties such as Top, Left, Width and Height. Often these designs have to represent fluid and responsive applications. Most web applications built from such designs use modern paradigms such as flex and grid display containers. The job of the Code generation is to get as close as possible to the application, a developer will build from a given design. This requires the Code generation to go through a set of heuristics and based on them to produce fluid design from the absolute/static one in Sketch.
```

Comment:
This requires the Code generation to go through a set of heuristics - To achieve that, the Code Generation applies a set of heuristics